framework/packages/abstract-client/src/account.rs

```diff
@@ -927,30 +961,6 @@ impl<Chain: CwEnv> Account<Chain> {
         }
     }
 
-    fn parse_modules_installing_response(response: Chain::Response) -> Addr {
-        let wasm_abstract_attributes: Vec<Attribute> = response
-            .events()
-            .into_iter()
-            .filter(|e| e.ty == "wasm-abstract")
-            .flat_map(|e| e.attributes)
-            .collect();
-
-        let module_addresses: String = wasm_abstract_attributes
-            .iter()
-            .find(|a| a.key == "new_modules")
-            .map(|a| a.value.parse().unwrap())
-            .unwrap();
-
-        // When there are multiple modules registered the addresses are returned in a common
-        // separated list. We want the last one as that is the "top-level" module while the rest
-        // are dependencies, since in the sub-account creation call, we pass in the top-level
-        // module last.
-        let module_address = module_addresses.split(',').last().unwrap();
-
-        // We install only one module
-        Addr::unchecked(module_address)
-    }
-
```

Comment:
This method is not useful in combination with removing duplicate modules from install list, so got replaced to the more reliable method